<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>550</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="MazeWidget" name="mazeWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>500</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>500</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="title">
         <string>Rows        Cols</string>
        </property>
        <widget class="QSpinBox" name="rowsSpinBox">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>61</width>
           <height>31</height>
          </rect>
         </property>
         <property name="suffix">
          <string/>
         </property>
         <property name="prefix">
          <string/>
         </property>
         <property name="minimum">
          <number>2</number>
         </property>
         <property name="maximum">
          <number>50</number>
         </property>
        </widget>
        <widget class="QSpinBox" name="colsSpinBox">
         <property name="geometry">
          <rect>
           <x>63</x>
           <y>20</y>
           <width>61</width>
           <height>31</height>
          </rect>
         </property>
         <property name="prefix">
          <string/>
         </property>
         <property name="minimum">
          <number>2</number>
         </property>
         <property name="maximum">
          <number>50</number>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="generateMaze">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>10000</width>
          <height>30</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Generate Maze</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="selectFile">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Select a file</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exportFile">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Export to file</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="title">
         <string>x1               y1</string>
        </property>
        <widget class="QSpinBox" name="x1">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>61</width>
           <height>31</height>
          </rect>
         </property>
         <property name="maximum">
          <number>4</number>
         </property>
        </widget>
        <widget class="QSpinBox" name="y1">
         <property name="geometry">
          <rect>
           <x>63</x>
           <y>20</y>
           <width>61</width>
           <height>31</height>
          </rect>
         </property>
         <property name="maximum">
          <number>4</number>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="title">
         <string>x2              y2</string>
        </property>
        <widget class="QSpinBox" name="x2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>61</width>
           <height>31</height>
          </rect>
         </property>
         <property name="maximum">
          <number>4</number>
         </property>
        </widget>
        <widget class="QSpinBox" name="y2">
         <property name="geometry">
          <rect>
           <x>63</x>
           <y>20</y>
           <width>61</width>
           <height>31</height>
          </rect>
         </property>
         <property name="maximum">
          <number>4</number>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="searchWayButton">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Finding a way</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>100</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MazeWidget</class>
   <extends>QWidget</extends>
   <header>mazewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
